docs/cli-mode.md

```diff
@@ -321,4 +354,27 @@ Flags:
 ```
 The ```--config``` flag can be used to provide the path to the env specific config folder that contains the ```serverConfig.json```, ```toolConfig.json```, and ```keywordConfig.json``` files with the details of the environment to which the resources should be imported. If the flag is not provided, the tool looks for the server configurations in the environment variables.
 
-The ```--inputDir``` flag can be used to provide the path to the local directory where the resource configuration files are stored. If the flag is not provided, the tool looks for the resource configuration files at the current working directory.
\ No newline at end of file
+The ```--inputDir``` flag can be used to provide the path to the local directory where the resource configuration files are stored. If the flag is not provided, the tool looks for the resource configuration files in the current working directory.
+
+## Supported resource types
+The tool supports the following resource types:
+
+### Applications
+The tool supports exporting and importing applications. The exported application configuration files can be found under the ```Applications``` folder in the local directory. If it is required to deploy a new application through the `import` command of the tool, the new file should be placed under the ```Applications``` folder in the local directory.
+
+Since the ```Console``` and ```My Account``` are read-only system applications in WSO2 Identity Server, if it is required to update these applications through the `import` command, add the following configurations to the ```deployment.toml``` file and restart the server.
+```
+[system_applications]
+read_only_apps = []
+```
+> **Caution:** Be cautious when updating the system applications: ```Console``` and ```My Account``` through the tool, since it will result in unexpected errors in these apps if edited incorrectly. It is recommended to exclude the ```Console```, ```My Account``` and the Management application created for the tool during normal usage, unless it is required to update them through the tool.
+
+### Identity providers
+The tool supports exporting and importing identity providers. The exported identity provider configuration files can be found under the ```IdentityProviders``` folder in the local directory. If it is required to deploy a new identity provider through the import command of the tool, the new file should be placed under the ```IdentityProviders``` folder in the local directory.
+
+The resident identity provider can also be exported into a file named ```LOCAL``` and can be updated by modifying the ```LOCAL``` file and using the import command.
+> **Caution:** Be cautious when updating the resident identity provider through the ```LOCAL``` file since it will result in unexpected errors in the server if edited incorrectly. It is recommended to exclude the ```LOCAL``` file during normal usage, unless it is required to update the resident identity provider through the tool.
```

Comment:
```suggestion

> **Caution:** Be cautious when updating the resident identity provider through the ```LOCAL``` file since it will result in unexpected errors in the server if edited incorrectly. It is recommended to exclude the ```LOCAL``` file during normal usage unless it is required to update the resident identity provider through the tool.
```